### PR TITLE
Ember 1.11 blog post

### PR DESCRIPTION
**Pull request by mixonic**
[View as HTML](https://github.com/mixonic/website/blob/blag/source/blog/2015-03-27-ember-1-11-0-released.md)

### CODE REVIEW

File: source/blog/2015-03-27-ember-1-11-0-released.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+However these bindings do not currently have an escaping strategy,
+and thus will warn in the console when used:
+
+```handlebars
+{{! No escaping strategy, log a warning }}
+<a style="width: {{someProperty}}px"></a>
+
+{{! No escaping strategy, log a warning }}
+<a onmouseover="alert('{{someProperty}}');"></a>
```
Comment (mixonic) [inline source/blog/2015-03-27-ember-1-11-0-released.md]: In reality, only `style` warns. This should be made explicit in the blog post, and the implication that JS bindings warn in 1.11 removed. However, we will likely introduce warnings in 1.12.

/cc @rwjblue @stefanpenner @tomdale

Comment (mixonic) [inline source/blog/2015-03-27-ember-1-11-0-released.md]: Via @rwjblue 

> Only changes in 1.11 will be style warning, and embed and base now except src and href

**Comment by mixonic** [comment]
Read to roll.

**Comment by stefanpenner** [comment]
LGTM

**Pull request merged**